easybuild/easyconfigs/w/Wannier90/wannier90-3.1.0-gofb-2020a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'wannier90'
```

Comment:
Needs to be named `Wannier90` (and the file too). The name must match the capitalization of the folder